### PR TITLE
Work page

### PR DESCRIPTION
**Pull request by felixDev22**
In this work page repo:

- I added the work/ portfolio section.
- I also added the about section.
-  I also updated the section tag to the main tag.
-  I wrapped the nav and phone display icons in the header.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -134,9 +77,179 @@ <h1>I’m Felix<br />Glad to see you!</h1>
...
+              <h3>Back End Dev</h3>
+              <span class="bullet-point"></span>
+               <h3>2015</h3>
+              </div>
+              <p>A daily selection of privately personalized reads; no accounts or<br>sign-ups required.</p>
+              <div class="work-tools">
+                <button type="button" class="tools-btn">html</button>
+                  <button type="button" class="tools-btn">css</button>
+                  <button type="button" class="tools-btn">JavaScript</button>
+              </div>
```
Comment (levy002) [inline index.html]: - [x] Kindly make sure to wrap your stack into the `ul` tag and wrap each one into the `li` tag in order to correctly format your HTML semantically. :wink: 
  ### Make sure to apply this change on all your project cards.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @felixDev22   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (felixDev22) [reply index.html]: Thanks @levy002

**Review by andgarzonmal — APPROVED** [review]
Hi @felixDev22, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 
- Nice work following the instructions from previous reviewer ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**